reverb/pip_package/build_pip_package.sh

```diff
@@ -71,6 +71,19 @@ function prepare_src() {
   # must remain where they are for TF to find them.
   find "${TMPDIR}/reverb/cc" -type d -name ops -prune -o -name '*.so' \
     -exec mv {} "${TMPDIR}/reverb" \;
+
+  # Copy darwin libs over so they can be loaded at runtime
```

Comment:
why does this need to be done differently for darwin vs. linux?  on linux you need to copy the so files over as well.

Reply:
inspired by https://github.com/tensorflow/tensorflow/blob/v2.4.1/tensorflow/tools/pip_package/build_pip_package.sh#L164, It fixes https://github.com/deepmind/reverb/pull/24#issuecomment-731569061